### ISSUE TITLE
Azure provider.tf generated is missing features block

### ISSUE THREAD

**Issue by rich-ehrhardt** [cloud-native-toolkit/iascable#171]
The provider block for Azure needs to include an empty features block.
Currently:
```
provider "azurerm" {

  subscription_id = var.subscription_id
  client_id = var.client_id
  client_secret = var.client_secret
  tenant_id = var.tenant_id
}
```

Should be:
```
provider "azurerm" {
  features { }

  subscription_id = var.subscription_id
  client_id = var.client_id
  client_secret = var.client_secret
  tenant_id = var.tenant_id
}
```

**Comment by rich-ehrhardt** [comment]
This causes terraform to fail when being run on generated output.

**Comment by seansund** [comment]
Fixed in v2.25.5
